torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -819,7 +819,18 @@ def encode_forward(
                     for key in feat
                 })
             else:
-                x = self.model_dict[col_name](feat[:, i])
+                input_feat = feat[:, i]
+                if isinstance(input_feat, MultiEmbeddingTensor):
+                    input_feat = input_feat.values
+
+                # Numerical case:
+                if input_feat.ndim == 1:
+                    input_feat = input_feat.view(-1, 1, 1)
+                # MultiNestedEmbedding case:
+                elif input_feat.ndim == 2:
+                    input_feat = input_feat.unsqueeze(1)
```

Comment:
I think this logic can be made more robust and cleaner.
For instance if `input_feat` is `MultiNestedTensor` or `dict` of `MultiNestedTensor`, then `.ndim` will give an error. Can you add all stypes in this PR and refactor with a cleaner logic?